submission/submission-webapp/src/itest/kotlin/ac/uk/ebi/biostd/itest/test/submission/query/SubmissionListApiTest.kt

```diff
@@ -114,6 +116,57 @@ internal class SubmissionListApiTest(private val tempFolder: TemporaryFolder) :
             assertThat(submissionList).hasSize(5)
         }
 
+        @Test
+        fun `submission with section title`() {
```

Comment:
I am not sure Rick about this one, I like it that we have a strong API test but also worry that they become too many and hard are to track. 

The problem is that right now,  is that we do not have that good enough unit test so this full end to end is what is saving as. (Not changes are expected in code for this comment)

Reply:
I'm not sure if there's actually something like too much testing, maybe it's worth reading about it. AFAIK, is ok to have a new integration test for each feature.

Reply:
not saying too much testing, maybe unbalanced testing, like testing at the wrong level. Like for example testing unit by integration test. But about "is ok to have a new integration test for each feature." it may be yes, just wondering best way to organize them and keep track of them